recipes/sidetable/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sidetable" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 44695ab38e7e166664bf79cf5af23f4360f4d9a5e17114b5ebc4a6908f8be3d9
+
+build:
+  number: 0
```

Comment:
This looks like it can be `noarch: python`

Reply:
Got it. Will do.